nilearn/interfaces/fmriprep/load_confounds.py

```diff
@@ -315,7 +327,31 @@ def load_confounds(
 def _load_confounds_for_single_image_file(
     image_file, strategy, demean, **kwargs
 ):
-    """Load confounds for a single image file."""
+    """Load confounds for a single image file.
+
+    Parameters
+    ----------
+    image_file : str
+        Path to processed image file.
+
+    strategy : tuple or list of strings.
+        See :func:`nilearn.interfaces.fmriprep.load_confounds` for details.
+
+    demean : boolean, default True
+        See :func:`nilearn.interfaces.fmriprep.load_confounds` for details.
+
+    kwargs : dict
+        Extra relevant parameters for the given `strategy`.
+        See :func:`nilearn.interfaces.fmriprep.load_confounds` for details.
+
+    Returns
+    -------
+    sample_mask : None, numpy.ndarray, or list of
```

Comment:
just to be sure this means :
- None, numpy.ndarray, or list of numpy.ndarray

Reply:
```suggestion
    sample_mask : None, :class:`numpy.ndarray`, or list of
```

Reply:
> just to be sure. 
> This means : "None, numpy.ndarray, or list of numpy.ndarray" ?

None, numpy.ndarray, or list of all of the above.
When the input is a list of images, the output may or may not have a sample mask depending on whether they need scrubbing or not etc

What would be a better way to explain this?

Reply:
```
 None, :class:`numpy.ndarray` or, :obj:`list` of :class:`numpy.ndarray` or None
```

Would this work?